frontend/src/libs/hooks/use-debounce/use-debounce.hook.ts

```diff
@@ -0,0 +1,20 @@
+import { useEffect, useState } from '../hooks.js';
+import { DEFAULT_DEBOUNCE_DELAY } from './libs/constants/constants.js';
+
+const useDebounce = <T>(value: T, delay?: number): T => {
```

Comment:
You can use prepared solution of debounce lib

Reply:
done